### DISCUSSION TITLE
How do you pronounce "zod"?

### DISCUSSION THREAD

**Discussion by hiro0218** [colinhacks/zod#2779] [Q&A] [answered]
I've encountered a slight issue regarding the pronunciation of "zod". While this may seem a minor point, I believe understanding the correct pronunciation will contribute to accurate communication when discussing the software in my community, which largely consists of non-native English speakers.

There's currently no phonetic guide or pronunciation key available in the software's documentation. Therefore, I kindly request some guidance on the matter. How should "Zod" be pronounced? If possible, could you provide this information in the International Phonetic Alphabet (IPA) or some other universally recognized format?

**Comment by midzdotdev** [comment, marked as answer]
I believe it's pronounced almost the same as "God" and "rod". I've heard Colin speak of it before on YouTube and this is how it sounds. Not very familiar with IPA sorry.

---
https://www.howtopronounce.com/zod

**Comment by MatinT-SA** [comment]
I wanted to chime in on the pronunciation of zod.

It's actually pronounced with a hard `Z` sound, like in "Zebra" or "Zorro", not with an `S` sound. So, it's "Zod" (/zɑːd/), rhyming with words like "god" or "rod."

This is different from libraries like zustand, where the `Z` is pronounced like `Ts` (following German pronunciation).

Hope this clears things up for everyone!"
